.eslintrc.cjs

```diff
@@ -709,13 +699,9 @@ const rules = {
       next: '*'
     }
   ],
-  '@typescript-eslint/quotes': 0,
   '@typescript-eslint/require-await': 1,
   '@typescript-eslint/return-await': 1,
-  '@typescript-eslint/semi': 0,
-  '@typescript-eslint/space-before-blocks': 0,
-  '@typescript-eslint/space-before-function-paren': 0,
-  '@typescript-eslint/space-infix-ops': 0,
+  '@typescript-eslint/use-unknown-in-catch-callback-variable': 1,
```

Comment:
https://typescript-eslint.io/rules/use-unknown-in-catch-callback-variable/

Reply:
Should we enable this as well?
https://www.typescriptlang.org/tsconfig/#useUnknownInCatchVariables